test/rules/file-header/good-allow-single-line-comments/tslint.json

```diff
@@ -0,0 +1,8 @@
+{
+    "rules": {
+      "file-header": [true, {
+        "match": "Good header",
+        "allowSingleLineComments": true
```

Comment:
none of our options use camelCase, we only use kebab-case

Reply:
Woops, I thought I saw an example of options using camelCase. Fixed.